<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseDlg</class>
 <widget class="QDialog" name="BaseDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>About Meet4Eat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/application.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#BaseDlg {
  border: 1px solid rgb(151,167, 187);
}
QWidget {
  color: white;
  background-color: rgb(28,52,62);
}
QLineEdit {
  background-color: rgb(101,117, 137);  
}
QPushButton {
  background-color: rgb(111,127, 147);
}
QPushButton:hover {
  background-color: rgb(151,167, 187);
}
QPushButton[enabled=&quot;false&quot;] {
  background-color: rgb(31,47,67);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,0,0">
   <item>
    <widget class="QWidget" name="widgetHead" native="true">
     <property name="cursor">
      <cursorShape>SizeAllCursor</cursorShape>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="font">
         <font>
          <family>Roboto</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClose">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(28,52,62);
  border: 1px solid rgb(131,147, 167);
  border-image: url(:/icon-close.png);
}
QPushButton:hover {
  border: 5px solid white;
  border-radius:  4px;
}</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLineHead">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetClientArea" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="labelLineBottom">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutFooter">
     <item>
      <spacer name="spacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Btn3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Btn2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Btn1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>BaseDlg</receiver>
   <slot>onBtnCloseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton1</sender>
   <signal>clicked()</signal>
   <receiver>BaseDlg</receiver>
   <slot>onBtn1Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton2</sender>
   <signal>clicked()</signal>
   <receiver>BaseDlg</receiver>
   <slot>onBtn2Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton3</sender>
   <signal>clicked()</signal>
   <receiver>BaseDlg</receiver>
   <slot>onBtn3Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnCloseClicked()</slot>
  <slot>onBtn1Clicked()</slot>
  <slot>onBtn2Clicked()</slot>
  <slot>onBtn3Clicked()</slot>
 </slots>
</ui>
